config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
```

Comment:
The commit message for this commit does not accurately describe the contents of the commit.